### PR TITLE
Update Google Cloud SDK to version 404 [VS-648]

### PR DESCRIPTION
**Pull request by mcovarr**
Freshening up and syncing with [pending changes in GVS](https://github.com/broadinstitute/gatk/pull/8044/files).

### CODE REVIEW

File: supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala [hunk 1]
```diff
@@ -27,17 +33,9 @@ object ActionUtils {
     *
     * FYI additional older versions are available on DockerHub at:
     * https://hub.docker.com/r/google/cloud-sdk
-    *
-    * When updating this value, also consider updating the CromwellImagesSizeRoundedUpInGB below.
```
Comment (mcovarr) [inline supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala:31]: `CromwellImagesSizeRoundedUpInGB` wasn't being used so I deleted it instead.

**Review by aednichols — APPROVED** [review]

File: supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala [hunk 2]
```diff
@@ -27,17 +33,9 @@ object ActionUtils {
...
     */
   val CloudSdkImage: String =
-    config.getOrElse("cloud-sdk-image-url", "gcr.io/google.com/cloudsdktool/cloud-sdk:354.0.0-alpine")
-
-  /*
-   * At the moment, cloud-sdk (584MB for 354.0.0-alpine) and stedolan/jq (182MB) decompressed ~= 0.8 GB
-   */
-  val CromwellImagesSizeRoundedUpInGB: Int =
-    config.getOrElse("cloud-sdk-image-size-gb", 1)
+    config.getOrElse("cloud-sdk-image-url", "gcr.io/google.com/cloudsdktool/cloud-sdk:404.0.0-alpine")
```
Comment (breilly2) [inline supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala:38]: I'm planning to experiment with the `slim` image at the suggestion of https://partnerissuetracker.corp.google.com/issues/247814958#comment6. I'm planning to run the experiment with `354.0.0-slim` and `404.0.0-alpine`, and possibly `404.0.0-slim` depending on the result of the other experiments.

Comment (mcovarr) [reply supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala:38]: Interesting, good luck with that issue!

File: supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala [hunk 3]
```diff
@@ -27,17 +33,9 @@ object ActionUtils {
...
     * FYI additional older versions are available on DockerHub at:
     * https://hub.docker.com/r/google/cloud-sdk
-    *
-    * When updating this value, also consider updating the CromwellImagesSizeRoundedUpInGB below.
     */
   val CloudSdkImage: String =
-    config.getOrElse("cloud-sdk-image-url", "gcr.io/google.com/cloudsdktool/cloud-sdk:354.0.0-alpine")
-
-  /*
-   * At the moment, cloud-sdk (584MB for 354.0.0-alpine) and stedolan/jq (182MB) decompressed ~= 0.8 GB
```
[the remark this answers is not in the document]
Comment (breilly2) [reply supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala:37]: You also removed the `cloud-sdk-image-size-gb` configuration value, which we should probably keep if we're going to keep the `cloud-sdk-image-url` config value... except it looks like, oops, we're only using `cromwellImagesSizeRoundedUpInGB` and not `CromwellImagesSizeRoundedUpInGB`. I'm guessing that's an oversight that didn't get caught when `CromwellImagesSizeRoundedUpInGB` was initially introduced.

**Comment by aednichols** [comment]
Superseded by https://github.com/broadinstitute/cromwell/pull/7359

**Pull request closed without merging**